pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -76,11 +77,30 @@ func (pl *CSILimits) Name() string {
 // failed by this plugin schedulable.
 func (pl *CSILimits) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
-		{Event: framework.ClusterEvent{Resource: framework.CSINode, ActionType: framework.Add}},
+		{Event: framework.ClusterEvent{Resource: framework.CSINode, ActionType: framework.Add}, QueueingHintFn: pl.isSchedulableAfterCSINodeAdded},
 		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}},
 	}
 }
 
+func (pl *CSILimits) isSchedulableAfterCSINodeAdded(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
Can we just simply return Queue for all CSINode/Add events? A new Node should be clean and no Pod should be there, right?

Reply:
> A new Node should be clean and no Pod should be there, right?

Right. But, according to my understanding, CSI nodes should be installed with the drivers that a scheduling pod needs. Feel free to correct me if I'm wrong 🙏

Reply:
According to the `Filter` implementation, it doesn't care such scenario where CSI node doesn't have a required driver, does it?

Reply:
I was very wrong! That's right. Should I leave a comment for code readers in such a case?

Reply:
Yes, please leave a comment that describes why we don't have QHint intentionally.

Reply:
Done!